modules/account-lib/src/coin/sol/iface.ts

```diff
@@ -55,3 +55,17 @@ export interface TransactionExplanation extends BaseTransactionExplanation {
   durableNonce?: DurableNonceParams;
   memo?: string;
 }
+
+export interface validityWindow {
```

Comment:
name for interface first char is in uppercase
ValidityWindow
SequenceId

Reply:
this should be included on this file:
https://github.com/BitGo/BitGoJS/blob/master/modules/account-lib/src/coin/baseCoin/iface.ts
Not in sol iface